pages/releases/4.3.html

```diff
@@ -0,0 +1,1904 @@
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#dotnet-more-robust-reload-process">More robust reload process</a>
+								</h4>
+								<div class="c-blockquote">
+									Issues have been <span class="highlight">fixed</span>
+								</div>
+								<div class="c-content">
+									<p>
+										One of the most annoying issues that C# users have with Godot have been fixed.
+									</p>
+									<p>
+										When a C# project is re-built, scripts need to be reloaded. Unfortunately, a
+										lot of things can go wrong during this process. It usually left users in an
+										invalid state with no other option but to re-open the Godot editor entirely.
+									</p>
+									<p>
+										Now, the process has been improved and users will be able to appreciate the new
+										robustness of the reloads.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="paulloz"
+									data-read-more="https://github.com/godotengine/godot/pull/83217">
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #dotnet-more-robust-reload-process -->
+
+				<div id="dotnet-better-support-generic-csharp-classes" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#dotnet-better-support-generic-csharp-classes">
+										<span>Better support of generic C# classes</span>
+									</a>
+								</h4>
+								<div class="c-blockquote">
+									Patience yields <span class="highlight">results</span>!
+								</div>
+								<div class="c-content">
+									<p>
+										This feature has been requested for a long time, and it's finally here.
+									</p>
+									<p>
+										While the implementation has some limitations, you'll be finally able to use
+										(more comfortably) generic C# classes in your code.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="raulsntos"
+									data-read-more="https://github.com/godotengine/godot/pull/88371">
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #dotnet-better-support-generic-csharp-classes -->
+
+				<div id="dotnet-preventing-export-without-csharp-code" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#dotnet-preventing-export-without-csharp-code">
+										<span>Preventing export without C# code</span>
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										The .solution and .csproj files are required for Godot to build C# projects.
```

Comment:
The file extension is `.sln`.

```suggestion
										The .sln and .csproj files are required for Godot to build C# projects.
```